app/src/main/java/br/com/catbag/gifreduxsample/MyApp.java

```diff
@@ -26,6 +30,9 @@ private void initializeFluxxan() {
         AppState state = ImmutableAppState.builder().build();
```

Comment:
Now we can extract a method, something like: `initializeFluxxan` or initializeReduxInfra`